src/teleop/gui/src/components/MotorAdjust.vue

```diff
@@ -0,0 +1,140 @@
+<!--
```

Comment:
This comment courtesy of github copilot, would be cool to do this for all vue files though